test/ext/type.test.js

```diff
@@ -224,4 +226,19 @@ describe('type', function() {
     }, 'expected Array [] to be an error',
        '    expected Array [] to be an instance of Error');
   });
+
+  it('test Date', function() {
+    (new Date()).should.be.a.Date();
+    (new Date(Date.now())).should.be.a.Date();
+    ('2016-01-06T12:22:03.269Z').should.be.a.Date();
+    ('2015-02-27').should.be.a.Date();
+    // but
+    should(new Date()).should.not.be.a.Date();
```

Comment:
That is pretty useless. `should(new Date())` returns `Assertion`, next you call `.should` on Assertion